content/en/agent/kubernetes/helm.md

```diff
@@ -138,9 +138,51 @@ datadog:
  logsConfigContainerCollectAll: true
 ```
 
+### Enable APM and Distributed Tracing
+
+Update your [datadog-values.yaml][7] file with the following APM configuration:
+
+```
+datadog:
+  (...)
+  apmEnabled: true
+
+(...)
+
+daemonset:
+  (...)
+  useHostPort: true
+```
+
+Use the downward API to pull the host IP; the application container needs an environment variable that points to `status.hostIP`. You can name it `DD_AGENT_HOST`. Update the `env` section of your `datadog-values.yaml` file with the following:
+
+```
+env:
+  - name: DD_AGENT_HOST
+    valueFrom:
+      fieldRef:
+        fieldPath: status.hostIP
+```
+
+Then upgrade your Datadog Helm chart.
+
+Finally, point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_AGENT_HOST`. For example, in Python:
```

Comment:
Similar to my previous comment [here](https://github.com/DataDog/documentation/pull/4352/files#r273689254). We can say: `Finally, point your application-level tracers to the host IP using the environment variable `DD_AGENT_HOST`